src/client/components/VotingBox.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+const VotingBox = ({ voteForItem, votesNumber, whetherUserVoted }) => (
+  <button
+    aria-label="Vote for item"
```

Comment:
do we need this? 
For now we are not supporting screen readers so I think it's not necessary